transport/ice/IceSession.h

```diff
@@ -292,28 +307,25 @@ class IceSession
         uint64_t pendingRequestAge(uint64_t now) const;
         size_t pendingTransactionCount() const;
 
+        IceSession* _iceSession;
         uint64_t _transmitInterval;
         int _replies;
         IceError _errorCode;
         uint64_t _minRtt;
-        const std::string& _name;
         std::deque<StunTransaction> _transactions; // TODO replace with inplace circular container
-        StunTransactionIdGenerator& _idGenerator;
-        const IceConfig& _config;
-        const SessionCredentials& _credentials;
-        crypto::HMAC& _hmacComputer;
```

Comment:
Reduce IceCandidatePair memory footprint. We store only a pointer to IceSession so we can have access fom everything we need